Personnummer/Personnummer.cs

```diff
@@ -24,7 +24,15 @@ public Options()
 
         public DateTime Date { get; private set; }
 
-        public int Age => DateTime.Now.Year - Date.Year;
+        public int Age
+        {
+            get
+            {
+                var now = DateTime.Now;
+                var differenceInTotalMonth = 12 * (now.Year - Date.Year) + now.Month - Date.Month;
+                return differenceInTotalMonth / 12;

```

Comment:
This will handle the month, but not the day, so we might diff on up to 30 days on a month.

Reply:
Very true, I somehow completely forget the days :D Will push an update soon.